src/components/MatomoOptOut.tsx

```diff
@@ -1,10 +1,11 @@
 import { useEffect, useState } from "react"
-import { Checkbox, Flex } from "@chakra-ui/react"
 
 import Text from "@/components/OldText"
```

Comment:
This one needs to be replaced by the native `<p>` tag

Reply:
Done and added note above.